### DISCUSSION TITLE
Simulate toaster

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6493] [Help] [answered]
I think we could simulate toaster message with the tag component, here in green :
![image](https://user-images.githubusercontent.com/28592069/176409582-2cc14681-4b9b-4d8a-9f0a-0cb1d2f7da82.png)

But how can I have the tag hidden by clicking on the little cross ?
I wanted to use the conditions...

**Comment by melohagan** [comment, marked as answer]
Hi @NicolasFabre81 

Have you tried setting App State (https://docs.budibase.com/docs/app-state) onClick of the X.
Then you can have your Show/Hide condition dependent on the app state binding.

**Reply by NicolasFabre81** [reply]
Perfect !
